### PR TITLE
875 feature request exif tool

### PR DESCRIPTION
**Pull request by SUA-321**
Completed Exif Tool and tested thoroughly, added code standardisation.

### CODE REVIEW

**Comment by basoreeper3007** [comment]
I will review later today. :)

**Review by basoreeper3007 — DISMISSED** [review]
Great work here @SUA-321! Please make the requested changes and I will approve it for merging. I tested the application and I'm happy with its functionality.

File: src/components/RouteWrapper.tsx [hunk 1]
```diff
@@ -428,6 +429,13 @@ export const ROUTES: RouteProperties[] = [
         description: "A tool used for capturing screenshots and gathering information from web servers",
         category: "Web Application Testing",
     },
+    {
+        name: "Exif",
+        path: "/tools/Exif",
+        element: <Exif />,
+        description: "A tool used to extract metadata from different file types",
+        category: "Information Gathering and Analysis",
+    },
```
Comment (basoreeper3007) [inline src/components/RouteWrapper.tsx:438]: Routewrapper tools should be in alphabetical order. Please place Exif above EyeWitness.

File: src/components/Exif/Exif.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    // Component Constants
+    const title = "ExifTool";
+    const description =
+        "ExifTool is a platform-independent command-line application for reading, writing, and editing metadata in a wide variety of file types.";
+    const steps =
+        "Step 1: Install relevant dependencies from start page\n" +
+        "Step 2: Have an image saved locally and specify file path in the form.\n" +
+        "Step 3: Enter a Metadata tag e.g.Author\n" +
+        "Step 4: Choose to either write or read metadata from the image" +
+        "Step 5: Run the tool!"; // Steps explaining how to run tool
```
Comment (basoreeper3007) [inline src/components/Exif/Exif.tsx:46]: Simplification of user guide. Also some fixes with punctuation and formatting.

```suggestion
        "Step 1: Specify the file path of an image on your machine.\n" +
        "Step 2: Enter a metadata tag to read or write to.\n" +
        "Step 3: Choose whether to read or write to the specified metadata field.\n" +
        "Step 4: Input a value to write to the specified metadata field.\n" +
        "Step 5: Run the tool!";

**Review by basoreeper3007 — CHANGES_REQUESTED** [review]
Great work, now please make the changes I have suggested above. As mentioned earlier I have conducted application testing on this tool.

**Review by basoreeper3007 — APPROVED** [review]
Great job @SUA-321! I have tested this tool and I approve it for merging @michaeljpigott.

**Review by michaeljpigott — CHANGES_REQUESTED** [review]
Hi @SUA-321 

Can you please explain why you need to use pkexec for this tool? 

It appears to run in the terminal without sudo: 

image

It also appears that the loading overlay is not working when I use the tool. 

I would have thought that you need to add exiftool as a scoped command in tauri.conf.json. 

Kind regards

Michael

Kind regards 

Michael

**Review by michaeljpigott — APPROVED** [review]
Hi @SUA-321 

I approve this pull request although when testing, the loading overlay still did not appear. No worries, good work and it looks like a good tool. 

Kind regards

Michael

**Pull request merged**